tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/tests/AbstractPitclipseSWTBotTest.java

```diff
@@ -132,6 +135,30 @@ public void run() {
         });
     }
 
+    public static void openViewById(String viewId) throws InterruptedException {
+        Display.getDefault().syncExec(()-> {
+            IWorkbench workbench = PlatformUI.getWorkbench();
+            try {
+                workbench.getActiveWorkbenchWindow().getActivePage().showView(viewId);
+            } catch (WorkbenchException e) {
+                e.printStackTrace();
+            }
+        });
+    }
+
+    public static void closeViewById(String viewId) {
+        final long previousTimeout = SWTBotPreferences.TIMEOUT;
+        try {
+            // set timeout short, because if the view is not present we don't wait long
+            SWTBotPreferences.TIMEOUT = 100;
```

Comment:
Please, iterate over the views using SWTBot and close the view if found.
Much better than this strategy, which is much less deterministic.